docs/whatsnew/2.0.rst

```diff
@@ -46,6 +46,11 @@ change has been made to behavior for Python 2.
 Other significant changes
 =========================
 
+Convolution has undergone a significant overhaul.  They keyword arguments have
```

Comment:
I would add a note here to clarify that the change is to make the two more consistent (rather than just being an arbitrary change).  Also maybe "Convolution" -> `` `~astropy.convolution` ``?